frontend/src/locales/pt/club.json

```
@@ -17,12 +18,20 @@
+      "terms_html": "Declaro que li e aceito os termos apresentados no <a href='/club-rules' class='underline'>Regulamento SGG Club</a>"
     },
+    "submit": "Aderir ao SGG Club",
     "titles": {
       "main": "Torne-se membro do Clube SGG"
     }
   },
+  "story": {
+    "not_found": "História não encontrada ou já não está disponível."
+  },
+  "stories": {
+    "subtitle": "Histórias Reais de Impacto e Esperança"
```

Biome (2.1.2)

[error] The key stories was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)